### PR TITLE
[OPERATOR-693] Create Operator 1.9.0 release branch

### PR DESCRIPTION
**Pull request by pure-jliao**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes # OPERATOR-693

**Special notes for your reviewer**:

### CODE REVIEW

**Review by haibinxie — APPROVED** [review]

File: deploy/olm-catalog/portworx/1.9.0/portworx-certified.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,540 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: portworx-operator.v1.8.0
```
Comment (piyush-nimbalkar) [inline deploy/olm-catalog/portworx/1.9.0/portworx-certified.clusterserviceversion.yaml:4]: 1.9.0

File: deploy/olm-catalog/portworx/1.9.0/portworx-certified.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,540 @@
...
+          "metadata": {
+            "name": "example",
+            "namespace": "test-operator"
+          },
+          "spec": {}
+        }
+      ]
+spec:
+  displayName: Portworx Enterprise
+  version: 1.8.0
```
Comment (piyush-nimbalkar) [inline deploy/olm-catalog/portworx/1.9.0/portworx-certified.clusterserviceversion.yaml:52]: 1.9.0

File: deploy/olm-catalog/portworx/1.9.0/portworx-certified.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,540 @@
...
+  minKubeVersion: "1.12.0"
+  maturity: stable
+  replaces: portworx-operator.v1.7.1
```
Comment (piyush-nimbalkar) [inline deploy/olm-catalog/portworx/1.9.0/portworx-certified.clusterserviceversion.yaml:55]: 1.8.0

**Review by piyush-nimbalkar — CHANGES_REQUESTED** [review]
- Makefile: Change the RELEASE_VER to 1.9.0
- Makefile: Change the BASE_REGISTRY_IMG to 1.8.0
- Change the image to 1.9.0-dev in `deploy/operator.yaml`

**Comment by piyush-nimbalkar** [comment]
We need to add the 1.9.0 CSV folder to master branch as well.

**Review by piyush-nimbalkar — APPROVED** [review]

**Pull request merged**